mantidimaging/gui/windows/recon/view.py

```diff
@@ -112,7 +111,10 @@ def on_row_change(item, _):
                 cor = row_data.cor
                 self.presenter.set_row(item.row())
                 self.presenter.set_last_cor(cor)
-                self.presenter.set_preview_slice_idx(slice_idx)
+                try:
+                    self.presenter.set_preview_slice_idx(slice_idx)
+                except Exception:
+                    pass
```

Comment:
Same - can we narrow down the exception, perhaps add a comment about why it's capturing it too

Reply:
Moved one layer lower in the stack, as this is not the best place to catch the exception. Same as above with astra throwing the exception as "Exception" will add the same comment.